lib/defra_ruby/area/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module DefraRuby
   module Area
-    VERSION = "2.1.0"
+    VERSION = "2.2.0"
```

Comment:
Just to confirm, this is minor due to the changes all being behind the scenes and not actually affecting the end user app?

Reply:
This is quite major. EA API version has been switched from 1.0.0 to 2.0.0, search filter has been fully rewritten, response format switched from XML to JSON. I've tried to make it backwards-compatible, although that's not guaranteed